configure

```diff
@@ -1350,6 +1350,36 @@ def configure_inspector(o):
                        options.without_ssl)
   o['variables']['v8_enable_inspector'] = 0 if disable_inspector else 1
 
+bin_override = None
+
+def setup_bin_overrides():
+  global bin_override
```

Comment:
What is the reason you use a global instead of returning a value?

Reply:
It mirrored what other stages were doing, with setting values in "o". Returning makes more sense, will change